lib/hal/led.h

```diff
@@ -0,0 +1,24 @@
+#ifndef HAL_LED_H
+#define HAL_LED_H
+
+typedef enum _LEDColor {
+  LED_OFF = 0x00,
+  LED_GREEN = 0x01,
+  LED_RED = 0x10,
+  LED_ORANGE = 0x11
+} LEDColor;
```

Comment:
nit: I wonder if we should prefix these somehow and if the style is right - it's part of the API, and once set, we shouldn't keep changing it.

Reply:
`FRONT_LED_XXX`?